.github/workflows/callable-e2e-runner.yml

```diff
@@ -35,19 +35,25 @@ jobs:
       framework: ${{ matrix.integ-config.framework }}
       category: ${{ matrix.integ-config.category }}
       sample_name: ${{ matrix.integ-config.sample_name }}
-      spec: ${{ matrix.integ-config.spec }}
+      spec: ${{ matrix.integ-config.spec || '' }}
       amplifyjs_dir: ${{ matrix.integ-config.amplifyjs_dir || false }}
       browser: |
```

Comment:
added extra use case
`browser is not provided by config`: send empty array`[""]`